classes/profile.php

```diff
@@ -63,6 +63,9 @@ class profile extends persistent {
     const SCRIPTTYPE_WS = 3;
     const SCRIPTTYPE_TASK = 4;
 
+    /** @var int Maximum length of string allowed into a char database field */
```

Comment:
So it's currently set to 256 but according to this https://docs.moodle.org/dev/XMLDB_column_types 255 seems to be the upper limit for older versions of MySQL. That may not be a factor anymore so assuming it's not, the comment should be updated as it seems to be describing all char fields within this plugin, though it's only being used by one in particular, the `parameters` field. 

I'd be okay with it being slightly more general e.g. if it was planned for other fields as well.
```suggestion
    /** @var int Maximum string length limit for longer char database fields */
```

Reply:
Changed DB field to TEXT